docs/guides/external-node/docker-compose-examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Running External node locally
+
+This directory contains two docker compose files for running external nodes, for sepolia testnet and mainnet.
+
+They are prepared to work out-of-the-box, no config changes are needed
+
+By default, the HTTP JSON-RPC API will run on port `3050`, while WS API will run on port `3051`.
```

Comment:
Aren't the ports actually 3060 and 3061?

Reply:
nice catch!, I looked at a wrong file